Version `0.14.0` cannot be compiled in projects not using `Finch`

Hello,

Since 0.14.0 `ex_vcr` cannot be compiled in projects that do not use `Finch`:

```
== Compilation error in file lib/exvcr/adapter/finch/converter.ex ==
Error: ** (CompileError) lib/exvcr/adapter/finch/converter.ex:67: Finch.Response.__struct__/0 is undefined, cannot expand struct Finch.Response. Make sure the struct name is correct. If the struct name exists and is correct but it still cannot be found, you likely have cyclic module usage in your code
    expanding struct: Finch.Response.__struct__/0
    lib/exvcr/adapter/finch/converter.ex:67: ExVCR.Adapter.Finch.Converter.response_to_string/1
could not compile dependency :exvcr, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile exvcr", update it with "mix deps.update exvcr" or clean it with "mix deps.clean exvcr"
Error: Process completed with exit code 1.
```

I think the module `ExVCR.Adapter.Finch.Converter` should also be wrapped in `if Code.ensure_loaded?(Finch)` like `ExVCR.Adapter.Finch`.

Thank you :)

--- a/lib/exvcr/adapter/finch/converter.ex
+++ b/lib/exvcr/adapter/finch/converter.ex
@@ -1,86 +1,91 @@
-defmodule ExVCR.Adapter.Finch.Converter do
-  @moduledoc """
-  Provides helpers to mock Finch methods.
-  """
+if Code.ensure_loaded?(Finch) do
+  defmodule ExVCR.Adapter.Finch.Converter do
+    @moduledoc """
+    Provides helpers to mock Finch methods.
+    """
 
-  use ExVCR.Converter
+    use ExVCR.Converter
 
-  alias ExVCR.Util
+    alias ExVCR.Util
 
-  defp string_to_response(string) do
-    response = Enum.map(string, fn({x, y}) -> {String.to_atom(x), y} end)
-    response = struct(ExVCR.Response, response)
+    defp string_to_response(string) do
+      response = Enum.map(string, fn {x, y} -> {String.to_atom(x), y} end)
+      response = struct(ExVCR.Response, response)
 
-    response =
-      if response.type == "error" do
-        body = string_to_error_reason(response.body)
-        %{response | body: body}
-      else
-        response
-      end
+      response =
+        if response.type == "error" do
+          body = string_to_error_reason(response.body)
+          %{response | body: body}
+        else
+          response
+        end
 
-    response =
-      if is_map(response.headers) do
-        headers = response.headers
-                  |> Map.to_list
-                  |> Enum.map(fn({k,v}) -> {k, v} end)
-        %{response | headers: headers}
-      else
-        response
-      end
+      response =
+        if is_map(response.headers) do
+          headers =
+            response.headers
+            |> Map.to_list()
+            |> Enum.map(fn {k, v} -> {k, v} end)
 
-    response
+          %{response | headers: headers}
+        else
+          response
+        end
+
+      response
+    end
+
+    defp string_to_error_reason(reason) do
+      {reason_struct, _} = Code.eval_string(reason)
+      reason_struct
+    end
+
+    defp request_to_string([request, finch_module]) do
+      request_to_string([request, finch_module, []])
+    end
+
+    defp request_to_string([request, _finch_module, opts]) do
+      url =
+        Util.build_url(request.scheme, request.host, request.path, request.port, request.query)
+
+      %ExVCR.Request{
+        url: parse_url(url),
+        headers: parse_headers(request.headers),
+        method: String.downcase(request.method),
+        body: parse_request_body(request.body),
+        options: parse_options(sanitize_options(opts))
+      }
+    end
+
+    # If option value is tuple, make it as list, for encoding as json.
+    defp sanitize_options(options) do
+      Enum.map(options, fn {key, value} ->
+        if is_tuple(value) do
+          {key, Tuple.to_list(value)}
+        else
+          {key, value}
+        end
+      end)
+    end
+
+    defp response_to_string({:ok, %Finch.Response{} = response}), do: response_to_string(response)
+
+    defp response_to_string(%Finch.Response{} = response) do
+      %ExVCR.Response{
+        type: "ok",
+        status_code: response.status,
+        headers: parse_headers(response.headers),
+        body: to_string(response.body)
+      }
+    end
+
+    defp response_to_string({:error, reason}) do
+      %ExVCR.Response{
+        type: "error",
+        body: error_reason_to_string(reason)
+      }
+    end
+
+    defp error_reason_to_string(reason), do: Macro.to_string(reason)
   end
-
-  defp string_to_error_reason(reason) do
-    {reason_struct, _} = Code.eval_string(reason)
-    reason_struct
-  end
-
-  defp request_to_string([request, finch_module]) do
-    request_to_string([request, finch_module, []])
-  end
-
-  defp request_to_string([request, _finch_module, opts]) do
-    url = Util.build_url(request.scheme, request.host, request.path, request.port, request.query)
-
-    %ExVCR.Request{
-      url: parse_url(url),
-      headers: parse_headers(request.headers),
-      method: String.downcase(request.method),
-      body: parse_request_body(request.body),
-      options: parse_options(sanitize_options(opts))
-    }
-  end
-
-  # If option value is tuple, make it as list, for encoding as json.
-  defp sanitize_options(options) do
-    Enum.map(options, fn({key, value}) ->
-      if is_tuple(value) do
-        {key, Tuple.to_list(value)}
-      else
-        {key, value}
-      end
-    end)
-  end
-
-  defp response_to_string({:ok, %Finch.Response{} = response}), do: response_to_string(response)
-
-  defp response_to_string(%Finch.Response{} = response) do
-    %ExVCR.Response{
-      type: "ok",
-      status_code: response.status,
-      headers: parse_headers(response.headers),
-      body: to_string(response.body)
-    }
-  end
-
-  defp response_to_string({:error, reason}) do
-    %ExVCR.Response{
-      type: "error",
-      body: error_reason_to_string(reason)
-    }
-  end
-
-  defp error_reason_to_string(reason), do: Macro.to_string(reason)
 end

